css/index.css

```
@@ -1 +1,391 @@
+    letter-spacing: 0.22rem;
+    line-height: 5;
+    font-size: 1.1rem;
+  }
+}
+.heading .table {
+  display: table;
+  margin: 0 auto;
+}
+.heading #horizontal-list {
+  display: inline;
+  font-family: Calibri;
+  font-size: 0.75rem;
+  color: #91AFA5;
+  letter-spacing: 0.1rem;
+  list-style: none;
+}
+@media (max-width: 800px) {
+  .heading #horizontal-list {
+    letter-spacing: 0.1rem;
+    font-size: 0.7rem;
+  }
+}
+@media (max-width: 500px) {
+  .heading #horizontal-list {
+    letter-spacing: 0.1rem;
+    font-size: 0.4rem;
+  }
+}
+.heading #horizontal-list li {
+  display: inline;
+}
+.heading li::before {
+  content: "•";
+  color: #c6e9dd;
+  display: inline-block;
+  width: 1rem;
+}
+.heading .border-2 {
+  margin: 20px auto;
+  width: 80%;
+  border: 2px solid #cae7eb77;
+  border-radius: 10px;
+}
+@media (max-width: 800px) {
+  .heading .border-2 {
+    width: 80%;
+  }
+}
+.intro {
+  text-align: center;
+  margin: 0 auto;
+  width: 60%;
+}
+.intro h3 {
+  border-bottom: 1px solid black;
+  font-weight: bold;
+  font-family: Calibri;
+  font-size: 1.67rem;
+  line-height: 3;
+  color: black;
+  letter-spacing: 0.12rem;
+}
+@media (max-width: 800px) {
+  .intro h3 {
+    letter-spacing: 0.1rem;
+    line-height: 3;
+    font-size: 1.38rem;
+  }
+}
+@media (max-width: 500px) {
+  .intro h3 {
+    letter-spacing: 0.1rem;
+    line-height: 3;
+    font-size: 1.29rem;
+  }
+}
+.intro p {
+  font-family: Noto Sans;
+  font-size: 0.75rem;
+  line-height: 2;
+  color: #585555;
+  letter-spacing: rem;
+}
+.main-content {
+  width: 100%;
+  display: flex;
+}
+@media (max-width: 500px) {
+  .main-content {
+    display: block;
+    margin: 0 auto;
+  }
+}
+.main-content .left-content {
+  width: 70%;
+  margin: 0 20px 0 30px;
+}
+@media (max-width: 500px) {
+  .main-content .left-content {
+    width: 100%;
+    margin: 0 auto;
+  }
+}
+.main-content .right-content {
+  width: 30%;
+  margin: 0 0px 0 40px;
+}
+@media (max-width: 500px) {
+  .main-content .right-content {
+    width: 100%;
+    margin: 0 auto;
+  }
+}
+.main-content h4 {
+  border-bottom: 1px solid black;
+  font-weight: bold;
+  font-family: Calibri;
+  font-size: 1.5rem;
+  line-height: 3;
+  color: black;
+  letter-spacing: 0.1rem;
+}
+@media (max-width: 800px) {
+  .main-content h4 {
+    letter-spacing: 0.1rem;
+    line-height: 3;
+    font-size: 1.25rem;
+  }
+}
+@media (max-width: 500px) {
+  .main-content h4 {
+    letter-spacing: 0.1rem;
+    line-height: 3;
+    font-size: 1.1rem;
+  }
+}
+.main-content h5 {
+  font-weight: bold;
+  font-family: Calibri;
+  font-size: 1.1rem;
+  line-height: 2;
+  color: black;
+  letter-spacing: 0.05rem;
+}
+.main-content h6 {
+  font-weight: bold;
+  font-style: italic;
+  font-family: Calibri;
+  font-size: 0.95rem;
+  line-height: 2;
+  color: black;
+  letter-spacing: rem;
+}
+.main-content address {
+  font-family: Noto Sans;
+  font-size: 0.75rem;
+  line-height: 2;
+  color: #585555;
+  letter-spacing: rem;
+}
```

```suggestion
.main-content h5 {
  font-weight: bold;
  font-family: Calibri;
  font-size: 1.1rem;
  line-height: 2;
  color: black;
  letter-spacing: 0.05rem;
}
.main-content h6 {
  font-weight: bold;
  font-style: italic;
  font-family: Calibri;
  font-size: 0.95rem;
  line-height: 2;
  color: black;
  letter-spacing: rem;
}
.main-content address {
  font-family: Noto Sans;
  font-size: 0.75rem;
  line-height: 2;
  color: #585555;
  letter-spacing: rem;
}
```
It would have been nice to make this section of the code a little more DRY.